### PR TITLE
update header counter with right result.

### PR DESCRIPTION
**Pull request by AllStackDev1**
Closes: #1707 

# Description
- Watch, Star, and Fork Count are displaying incorrectly

Image

Simulator Screen Shot - iPhone 14 - 2023-03-30 at 05 33 59

### CODE REVIEW

**Review by hdJerry — CHANGES_REQUESTED** [review]
Something might have been broken.
Screenshot 2023-03-30 at 10 33 34

File: expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx [hunk 1]
```diff
@@ -6,25 +6,27 @@ import StarIcon from '../Icons/StarIcon';
...
 const RepoActionButtons = () => {
   const { width } = useWindowDimensions();
+  const { info } = useRepoInfoStore();
 
   const btns = [
     {
       text: 'Watch',
       Icon: EyeIcon,
-      count: 2,
+      count: info.watcherCount,
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx:19]:
```suggestion
      count: info?.watcherCount,
```

File: expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx [hunk 2]
```diff
@@ -6,25 +6,27 @@ import StarIcon from '../Icons/StarIcon';
...
     },
     {
       text: 'Star',
       Icon: StarIcon,
-      count: 2,
+      count: info.stargazerCount,
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx:24]:
```suggestion
      count: info?.stargazerCount,
```

File: expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx [hunk 3]
```diff
@@ -6,25 +6,27 @@ import StarIcon from '../Icons/StarIcon';
...
     },
     {
       text: 'Fork',
       Icon: GitBranchIcon,
-      count: 2,
+      count: info.forkCount,
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx:29]:
```suggestion
      count: info?.forkCount,
```

**Review by hdJerry — CHANGES_REQUESTED** [review]
This should fix the issue

File: expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx [hunk 4]
```diff
@@ -6,25 +6,27 @@ import StarIcon from '../Icons/StarIcon';
...
 const RepoActionButtons = () => {
   const { width } = useWindowDimensions();
+  const { info } = useRepoInfoStore();
 
   const btns = [
     {
       text: 'Watch',
       Icon: EyeIcon,
-      count: 2,
+      count: info?.watcherCount,
```
Comment (AllStackDev1) [inline expo-zustand-styled-components/src/components/RepoActionButtons/RepoActionButtons.tsx:19]:
```suggestion
      count: info?.watcherCount || 0,
```

**Review by hdJerry — APPROVED** [review]

**Pull request merged**